proto/src/control.rs

```diff
@@ -88,10 +88,10 @@ pub enum ServerSortResultCode {
     Other = 80,
 }
 
-impl TryFrom<u8> for ServerSortResultCode {
+impl TryFrom<&u8> for ServerSortResultCode {
     type Error = LdapProtoError;
 
-    fn try_from(value: u8) -> Result<Self, Self::Error> {
+    fn try_from(value: &u8) -> Result<Self, Self::Error> {

```

Comment:
note: Typically, `TryFrom<u8>` is preferred over `TryFrom<&u8>` (`u8` being a `Copy` type)